ingenialink/bin/FOE/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.2.0.2"
```

Comment:
must be updated to an official release before releasing

Reply:
Yes, it will be updated when the FoE application is released